infra/stacks/cloud-storage-service/index.ts

```diff
@@ -346,7 +340,7 @@ const cloudStorageService = new CloudStorageService(
       },
       {
         name: 'SYNC_SERVICE_URL',
-        value: `https://sync-service-${stack === 'dev' ? 'dev3' : stack}.macroverse.workers.dev`,
+        value: getServiceUrl(ServiceUrl.SYNC_SERVICE_URL),
```

Comment:
Why not DRY these, along with defining name & value to be those in the URL config, by doing what was done in the other PR: https://github.com/macro-inc/macro/pull/604/files#diff-dfc91b0cf37097880ee96b9ac045b0130f17689c648c3a7ae9bb27b4b7f67a76

This gives us global name for the URL, and makes it harder to make a typo.